build_tfm.py

```diff
@@ -71,7 +71,7 @@ def _clone_tfm_repo(commit):
     Clone TF-M git repos and it's dependencies
     :param commit: If True then commit VERSION.txt
     """
-    check_and_clone_repo("trusted-firmware-m", "latest-tfm", TF_M_BUILD_DIR)
+    check_and_clone_repo("trusted-firmware-m", "released-tfm", TF_M_BUILD_DIR)
```

Comment:
Commit message "Build TF-M v1.3.0 release" seems like a misnomer. It's more like "Build the most recently released version of TF-M"